weave/trace_server/current_calls_query.sql

```diff
@@ -0,0 +1,203 @@
+-- This is the current call stream query
+
+-- We get the lightly filtered call ids
```

Comment:
you will likely need to modify / add to `weave/trace_server/tests/test_calls_query_builder.py` which will allow you to more formally define your sql expectations